doc/source/getting_started/install.rst

```diff
@@ -270,6 +270,12 @@ For example, :func:`pandas.read_hdf` requires the ``pytables`` package, while
 optional dependency is not installed, pandas will raise an ``ImportError`` when
 the method requiring that dependency is called.
 
+Optional pandas dependencies can be managed as extras (e.g.,``pandas[performance, aws]>=1.5.0``)
+in a requirements.txt, setup or pyproject.toml file.
+Available optional dependencies are ``[all, performance, computation, aws,
```

Comment:
I think it would be necessary to document which dependency is included in each section, so it would be good to refactor the headers and dependencies below to reflect the new categories (except `all` which should implicitly contain all the dependencies listed here)

Reply:
Very good idea.  Done.